.github/workflows/ci.yml

```diff
@@ -42,8 +42,12 @@ jobs:
           # If the event is push to branch use the default ref.
           # If the event is pull request (`pull_request_target` in our case) use merge commit as ref to run lint over the PR's code.
 
-      - name: Hermit Environment
-        uses: ./.github/actions/hermit
+      - name: Initialize hermit
```

Comment:
can you clarify please why this change is needed? i'd assume that sharing this step is preferable? (caching mostly)